registry/directory/directory.go

```diff
@@ -104,72 +104,129 @@ func (dir *RegistryDirectory) subscribe(url *common.URL) {
 }
 
 // Notify monitor changes from registry,and update the cacheServices
-func (dir *RegistryDirectory) Notify(event *registry.ServiceEvent) {
-	go dir.update(event)
+func (dir *RegistryDirectory) Notify(event ...*registry.ServiceEvent) {
+	go dir.update(event...)
 }
 
 // update the cacheServices and subscribe service from registry
-func (dir *RegistryDirectory) update(res *registry.ServiceEvent) {
-	if res == nil {
-		return
+func (dir *RegistryDirectory) update(events ...*registry.ServiceEvent) {
+	for _, event := range events {
+		logger.Debugf("registry update, result{%s}", event)
+		logger.Debugf("update service name: %s!", event.Service)
```

Comment:
I think should move this block to line 142?

https://github.com/apache/dubbo-go/pull/741/files#diff-f1103e9118b5d7dcaaa8b1dac15314d7R142

Reply:
done.